README.md

```diff
@@ -157,6 +157,49 @@ CONFIG:
   pooling_api: http://vmpooler.delivery.puppetlabs.net/
 ```
 
+### Two hosts with arbitrary host settings with aribitrary lists
+
+```
+$ beaker-hostgenerator centos6-64m{hypervisor=none\,hostname=static-master\,disks=\[16\]-redhat7-64a{disks=\[8\,16\]}
```

Comment:
I'd simplify this so the new feature stands out more, so I'd probably remove the `hypervisor` and `hostname` settings from this